charts/akeyless-secure-remote-access/values.yaml

```diff
@@ -176,7 +176,11 @@ ztbConfig:
     maxReplicas: 14
     cpuAvgUtil: 50
     memAvgUtil: 50
-  
+
+  # Optional, provide a key-name to extract the username (value) of that key in sub-claims. The extracted username will be used to authenticated to the remote RDP target.
+  # This overrides (for Zero Trust Bastion only) the global setting with the same name
+  usernameSubClaim:
```

Comment:
Where's the rdp subclaim?

Reply:
This is it, the new, only for rdp, subclaim. under ztbConfig section.

Reply:
if both `usernameSubClaim` are depend on each other, then it might be better to put them under the "Global" section with a nested structure:

`usernameSubClaim:
    ztb:
    ssh:
`

Reply:
And have the config for ztb and ssh split from their main config to be also in this new nested structure?
If so, I can't change the current location of usernameSubClaim for BC.
I can add a structure under it or just add under it the 2 parameters with unique names (rdpUsernameSubClaim & sshUsernameSubClaim)